pkg/webhooks/xgboost/xgboostjob_webhook.go

```diff
@@ -71,10 +72,10 @@ func (w *Webhook) ValidateDelete(context.Context, runtime.Object) (admission.War
 
 func validateXGBoostJob(job *trainingoperator.XGBoostJob) field.ErrorList {
 	var allErrs field.ErrorList
-
```

Comment:
nit: revert formatting change to minimize distractions

Reply:
good point